docs/BackupNoSQL.md

```diff
@@ -0,0 +1,153 @@
+scalardb-headless                ClusterIP      None             <none>        60051/TCP                         2m2s
+scalardb-metrics                 ClusterIP      10.111.213.194   <none>        8080/TCP                          2m2s
+scalardl-auditor-envoy           LoadBalancer   10.111.141.43    <pending>     40051:31553/TCP,40052:31171/TCP   99s
+scalardl-auditor-envoy-metrics   ClusterIP      10.104.245.188   <none>        9001/TCP                          99s
+scalardl-auditor-headless        ClusterIP      None             <none>        40051/TCP,40053/TCP,40052/TCP     99s
+scalardl-auditor-metrics         ClusterIP      10.105.119.158   <none>        8080/TCP                          99s
+scalardl-ledger-envoy            LoadBalancer   10.96.239.167    <pending>     50051:32714/TCP,50052:30857/TCP   109s
+scalardl-ledger-envoy-metrics    ClusterIP      10.97.204.18     <none>        9001/TCP                          109s
+scalardl-ledger-headless         ClusterIP      None             <none>        50051/TCP,50053/TCP,50052/TCP     109s
+scalardl-ledger-metrics          ClusterIP      10.104.216.189   <none>        8080/TCP                          109s
+```
+
+### Pause
+
+You can send the pause request to Scalar product pods on the Kubernetes environment.
+
+* Example
+  * ScalarDB Server
+    ```console
+    kubectl run scalar-admin-pause --image=ghcr.io/scalar-labs/scalar-admin:<tag> --restart=Never -it -- -c pause -s _scalardb._tcp.<helm release name>-headless.<namespace>.svc.cluster.local
+    ```
+  * ScalarDL Ledger
+    ```console
+    kubectl run scalar-admin-pause --image=ghcr.io/scalar-labs/scalar-admin:<tag> --restart=Never -it -- -c pause -s _scalardl-admin._tcp.<helm release name>-headless.<namespace>.svc.cluster.local
+    ```
+  * ScalarDL Auditor
+    ```console
+    kubectl run scalar-admin-pause --image=ghcr.io/scalar-labs/scalar-admin:<tag> --restart=Never -it -- -c pause -s _scalardl-auditor-admin._tcp.<helm release name>-headless.<namespace>.svc.cluster.local
+    ```
+
+### Unpause
+
+You can send the unpause request to Scalar product pods on the Kubernetes environment.
+
+* Example
+  * ScalarDB Server
+    ```console
+    kubectl run scalar-admin-unpause --image=ghcr.io/scalar-labs/scalar-admin:<tag> --restart=Never -it -- -c unpause -s _scalardb._tcp.<helm release name>-headless.<namespace>.svc.cluster.local
+    ```
+  * ScalarDL Ledger
+    ```console
+    kubectl run scalar-admin-unpause --image=ghcr.io/scalar-labs/scalar-admin:<tag> --restart=Never -it -- -c unpause -s _scalardl-admin._tcp.<helm release name>-headless.<namespace>.svc.cluster.local
+    ```
+  * ScalarDL Auditor
+    ```console
+    kubectl run scalar-admin-unpause --image=ghcr.io/scalar-labs/scalar-admin:<tag> --restart=Never -it -- -c unpause -s _scalardl-auditor-admin._tcp.<helm release name>-headless.<namespace>.svc.cluster.local
+    ```
+
+### Check the pause completed time and unpause started time
```

Comment:
```suggestion
### Check the pause completed time and unpause starting time
```

Reply:
I used the words `pause completed` and `unpause started` based on the output of the `scalar-admin` command.

```
$ kubectl run scalar-admin-pause --image=ghcr.io/scalar-labs/scalar-admin:1.2.0 --restart=Never -it -- -c pause -s _scalardb._tcp.scalardb-headless.default.svc.cluster.local
Pause completed at 2023-03-10T05:33:55.503Z UTC (1678426435503)
```
```
$ kubectl run scalar-admin-unpause --image=ghcr.io/scalar-labs/scalar-admin:1.2.0 --restart=Never -it -- -c unpause -s _scalardb._tcp.scalardb-headless.default.svc.cluster.local
Unpause started at 2023-03-10T05:34:13.006Z UTC (1678426453006)
```

This is because users can know what `pause started time` and `unpause started time` refers to.
In other words, it is easier for users to know what values they should see, if the document's word and the command's output are the same.

However, it seems that `unpause starting` is better from the English/documentation perspective.
In this case, which word should we prioritize?
English correctness is a high priority? We should use the same word of command output?
(Or, should we update the output message of the command?)

Sorry, I don't know much about what should we do in this case from the documentation perspective.
What do you think?